SPECS/syslog-ng/syslog-ng.spec

```diff
@@ -149,6 +151,9 @@ fi
 %{_libdir}/pkgconfig/*
 
 %changelog
+* Fri May 09 2025 Kanishk Bansal <kanbansal@microsoft.com> - 3.33.2-8
+- Patch CVE-2024-47619
+
 * Mon Nov 27 2023 Saul Paredes <saulparedes@microsoft.com> - 3.33.2-7
```

Comment:
@Kanishk-Bansal can you make an attempt at fixing the pTest?  There's no %check section here.  Can you see if there is one available, maybe look at Fedora's SPEC file?

Reply:
Since we're talking about ptests for this spec, [here's a bug](https://microsoft.visualstudio.com/OS/_workitems/edit/47882516) from a while ago, which should provide some initial help in trying to figure out how to re-enable the tests.

Reply:
After further discussion, we will proceed without the test.